life.html

```diff
@@ -1 +1 @@
-<canvas id=G><svg onload=setInterval('for(h=[i=(G.width=N=75)*N];i--;G.getContext`2d`.fillRect(i%N,~j,1,h[i]=g?S-5?S==6:g[i]:Date.now()&1))for(z=S=9;z--;S-=g[~j%N*N+~(-i-z/3-N)%N])j=-i/N-z%3-N;g=h',g=0)>
+<canvas id=G><svg onload=setInterval('for(h=[i=(G.width=N=75)*N];i--;G.getContext`2d`.fillRect(i%N,~j,1,h[i]=g?S-5?S==6:g[i]:newDate&1))for(z=S=9;z--;S-=g[~j%N*N+~(-i-z/3-N)%N])j=-i/N-z%3-N;g=h',g=0)>
```

Comment:
The differ isn't great at always showing character VT apparently. If use `View file` you will see it.